scripts/elasticsearch-test-helpers/destination_connector/test-ingest-elasticsearch-output.py

```diff
@@ -39,32 +38,96 @@ def query(client: Elasticsearch, search_text: str):
     return client.search(index=INDEX_NAME, body=query)
 
 
-if __name__ == "__main__":
-    print(f"Checking contents of index" f"{INDEX_NAME} at {CLUSTER_URL}")
-
-    print("Connecting to the Elasticsearch cluster.")
-    client = Elasticsearch(CLUSTER_URL, basic_auth=(USER, PASSWORD), request_timeout=30)
-    print(client.info())
-
+def validate_count(client: Elasticsearch, num_elements: int):
+    print(f"Validating that the count of documents in index {INDEX_NAME} is {num_elements}")
     count = int(client.cat.count(index=INDEX_NAME, format="json")[0]["count"])
-    try:
-        assert count == N_ELEMENTS
-    except AssertionError:
-        sys.exit(
-            "Elasticsearch dest check failed:"
-            f"got {count} items in index, expected {N_ELEMENTS} items in index."
-        )
+    consistent = False
```

Comment:
What was the motivation for this? We were getting different counts?

Reply:
If there's anything surrounding chunking that affects this, it is no longer a static count. I did find that this count changed based on fixes to things like the chunker but this test is meant to isolate only the destination connector. These changes made it a bit more dynamic and based on the results that came out of the stager step in the pipeline.